labs/06-volumes.md

```diff
@@ -113,10 +118,27 @@ Multiple containers can attach to the same volume with data. Docker doesn't hand
 
 Let's try to go in and make a new html page for nginx to serve. We do this by making a new ubuntu container that has the `data` volume attached to `/tmp`, and thereafter create a new html file with the `echo` command:
 
-```bash
-docker run -ti --rm -v data:/tmp ubuntu bash
-root@9c36fcfcc048:# echo "<html><h1>hello world</h1></html>" > /tmp/hello.html
-root@9c36fcfcc048:# ls /tmp
+Start the container:
+
+```
```

Comment:
bash in the code snippet format?

Reply:
Sure, but would rather be consistent, the base annotation can create some weird syntax highlighting when mixing docker run commands and actual shell commands after that :shrug: